pypiper/manager.py

```diff
@@ -437,7 +436,7 @@ def _completed(self):
         :return bool: Whether the managed pipeline is in a completed state.
         """
         return (
-            self.pipestat.get_status(self._pipestat_manager.sample_name)
+            self.pipestat.get_status(self._pipestat_manager.cfg["record_identifier"])
```

Comment:
should be able to use `self._pipestat_manager.record_identifier` now that the properties are fixed.